packages/text-field/src/TextFieldPresenter.js

```diff
@@ -92,21 +96,26 @@ export default class TextFieldPresenter extends Component {
 
   static defaultProps = {
     id: generatedId(),
-    type: "text"
+    type: "text",
+    stayFocused: false
   };
 
   state = {
-    focused: false
+    focused: this.props.stayFocused
   };
 
   handleFocus = event => {
-    this.setState({ focused: true });
-    if (this.props.onFocus) this.props.onFocus(event);
+    if (!this.props.stayFocused) {
```

Comment:
Presentational logic shouldn't affect business logic. Regardless of whether the text field appears to be focused or blurred, the logic that determines whether it actually has focus should be remain the same. The changes here should be reverted to maintain a good separation of concerns.